analytics-listeners/src/main/java/org/exoplatform/analytics/listener/social/AnalyticsProfileListener.java

```diff
@@ -51,6 +51,7 @@ private StatisticData buildStatisticData(String operation, String username) {
     statisticData.setOperation(operation);
     statisticData.setUserId(getCurrentUserIdentityId());
     statisticData.addParameter(FIELD_SOCIAL_IDENTITY_ID, getUserIdentityId(username));
+    statisticData.addParameter("userCreatedDate", getUserCreatedDate(username));
```

Comment:
COuld you combine both calls in one ? We do not need to call the IdentityManager twice to load the Identity of the user